@xen-orchestra/lite/public/manifest.webmanifest

```
@@ -0,0 +1,14 @@
+{
+  "name": "XO Lite",
+  "short_name": "XOLite",
+  "start_url": "/",
+  "display": "standalone",
+  "description": "Integrated local and lightweight solution to manage a local XCP-ng pool.",
+  "icons": [
+    {
+      "src": "/publc/favicon.svg",
```

```suggestion
      "src": "favicon.svg",
```

Also, be careful to typo (`publc`).